### DISCUSSION TITLE
Foundry install

### DISCUSSION THREAD

**Discussion by learnsol** [foundry-rs/foundry#1464] [General]
Building forge(bin) step seems to be hanging 

while running below

git clone https://github.com/foundry-rs/foundry
cd foundry
cargo install --path ./cli --bins --locked --force

**Comment by onbjerg** [comment]
It's just really slow because it uses a lot of procedural macros. Just wait it out, depending on your computer it may take several minutes.

Alternatively you can get pre-built binaries for macOS (x86 and M1) and Linux (x86 and ARM) by using `foundryup` instead: https://book.getfoundry.sh/getting-started/installation.html#on-linux-and-macos
